incubator/kubeless/README.md

```diff
@@ -43,37 +43,47 @@ $ helm delete my-release
 
 The command removes all the Kubernetes components associated with the chart and deletes the release.
 
+## Kafka Trigger
+
+Kubeless supports triggering functions via kafka events. More info here: https://kubeless.io/docs/use-existing-kafka/.
```

Comment:
it would be useful to have instructions here of how to integrate this chart with the oficial Kafka (and zookeeper? ) charts.

Reply:
It is important to know that the property `kafkaBrokers` needs to be set